TESTING.md

```diff
@@ -55,18 +66,25 @@ Change the Spatial viewport:
 (For me, it's a two finger swipe up and down on trackpad.)
 
 Change the Spatial layers:
-- Hover over "Layers": The menu should become opaque.
-- Starting at the top, turn off each layer in turn, and you are left with just a black view.
-- Turn "cells" back on: You should have an arc of small grey blobs. (Or rectangles, if you zoom in enough.)
+- On the lefthand side, use the Layer Controller to change the opacity of each layer -  molecules should
+slowly disappear while the cell boundaries should remain as opacity decreases.
+- Turn on and off the molecules and cell segmentations layers.
+- In the image layer, change the colormap, domain, and opacity - confirm that each works as expected.
+  - The colormap should change the image so that both channels are in one colormap.
+  - The domain should change considerably how the sliders display but not how the image renders.
+  - Opacity should slowly decrease the opacity of whatever is shown in the image.
+- Change the channels and try adding another channel as well as removing channels.
+- Remove the image layer and then add it back.
 
 Hover:
 - As you move the mouse over different cells, the "Status" component should update.
+- Tooltips should appear over all other components that show data except for the one on which you are hovering.
```

Comment:
```suggestion
- Tooltips should appear over all other components that show data except for the one on which you are hovering.
- Where the tooltip does not appear, crosshairs should appear instead.
- Hover over a cell in the scatterplot and confirm that the crosshair appears over a cell with the same color in the spatial plot.
- Hover over a cell in the spatial plot and confirm that the crosshair appears over a cell with the same color in the scatterplot.
```

I am suggesting this task since in the past we have found bugs related to the crosshair / tooltip being flipped across the x- or y-axis.